packages/frontend/src/components/DownloadCsvButton/index.tsx

```diff
@@ -1,29 +1,26 @@
 import { Button } from '@blueprintjs/core';
+import { ResultRow } from '@lightdash/common';
 import { FC, memo } from 'react';
-import CSVExporter from '../CSVExporter';
 
 type Props = {
-    fileName: string | undefined;
-    rows: object[] | undefined;
+    rows: ResultRow[] | undefined;
```

Comment:
we could change this to `disabled: boolean` since we don't need the actual rows data anymore.